Tests/SonarScanner.MSBuild.Tasks.UnitTests/GetAnalyzerSettingsTests.cs

```diff
@@ -185,7 +185,7 @@ public void GetAnalyzerSettings_ConfigExists_Legacy_SettingsOverwritten()
         }
 
         [TestMethod]
-        public void GetAnalyzerSettings_ConfigExists_NewBehaviour_SettingsMerged()
+        public void ConfigExists_NewBehaviour_SettingsMerged()
```

Comment:
same problem as in [this comment](https://github.com/SonarSource/sonar-scanner-msbuild/pull/986#discussion_r608454213) ... NewBehaviour I imagine it means it's not "Legacy". But later on, maybe some "newer behavior" will be added. 

Could you remove "New Behaviour"?